gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/experimental/uikit/internal/configureInstallIosDeployTask.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.compose.experimental.uikit.internal
+
+import org.gradle.api.Project
+import org.jetbrains.compose.desktop.application.internal.MacUtils
+import org.jetbrains.compose.desktop.application.internal.UnixUtils
+import org.jetbrains.compose.experimental.uikit.tasks.AbstractComposeIosTask
+
+internal const val TASK_INSTALL_IOS_DEPLOY = "iosInstallIosDeploy"
+internal val Project.iosDeployExecutable get() = iosDeploySrc.resolve("build/Release/ios-deploy")
```

Comment:
Not sure if "Release" shall be hardcoded here.

Reply:
The ios-deploy utility is compiled from source. And in this directory the assembled binary is placed. All ios-deploy build logic is described in this repository https://github.com/ios-control/ios-deploy
"Release" means release version of binary ios-deploy